cabal.project

```diff
@@ -1,3 +1,4 @@
 packages:   streamly.cabal
           , test/streamly-tests.cabal
           , benchmark/streamly-benchmarks.cabal
+          , docs/streamly-docs.cabal
```

Comment:
`test` vs `benchmark` vs `docs`

singular vs plural

Reply:
`documentation` vs `documents (docs)`

Reply:
This came to my mind as well. I thought we can change the test/benchmark directory names to plural later to make everything consistent.